src/domains/citizen-auth-common/03_storage.tf

```
@@ -277,3 +277,25 @@ resource "azurerm_storage_queue" "profiles_to_sanitize" {
   name                 = "profiles-to-sanitize"
   storage_account_name = module.io_citizen_auth_storage.name
 }
+
+
+module "azure_storage_account" {
+  source = "github.com/pagopa/dx//infra/modules/azure_storage_account?ref=main"
+
+  environment                          = local.itn_environment
+  resource_group_name                  = data.azurerm_resource_group.data_rg_itn.name
+  tier                                 = "l"
+  subnet_pep_id                        = data.azurerm_subnet.subnet_pep_itn.id
+  private_dns_zone_resource_group_name = "${local.prefix}-${local.env_short}-rg-common"
```

Since we have the common_values module, we can get the resource group name straight from it

```suggestion
  private_dns_zone_resource_group_name = module.common_values.resource_groups.weu.common
```